ci/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
+# Install build dependencies, run unit tests and installed tests.
+
+set -xeuo pipefail
+
+dn=$(dirname $0)
+. ${dn}/libbuild.sh
+
+install_builddeps ostree
+
+dnf install -y sudo which attr fuse gjs parallel coccinelle clang \
+    libubsan libasan libtsan PyYAML gnome-desktop-testing redhat-rpm-config \
+    elfutils ${CI_PACKAGES:-}
```

Comment:
Any reason for using `CI_PACKAGES` instead of just plopping the extra ones directly in the YAML?